zebra-scan/src/init.rs

```diff
@@ -55,7 +55,7 @@ pub fn spawn_init(
         )
     } else {
         // TODO: spawn an entirely new executor here, to avoid timing attacks.
-        tokio::spawn(
+        tokio::task::spawn_local(
```

Comment:
I had to downgrade to single thread everywhere because of errors of this kind:

```
`(dyn ScanningKeyOps<SaplingDomain, AccountId, sapling_crypto::Nullifier> + 'static)` cannot be shared between threads safely
the trait `Sync` is not implemented for `(dyn ScanningKeyOps<SaplingDomain, AccountId, sapling_crypto::Nullifier> + 'static)`, which is required by `Instrumented<{async block@zebra-scan/src/init.rs:59:13: 66:14}>: std::marker::Send`
```

I am not sure how to solve it yet, it might need changes upstream.

Reply:
This problem was fixed in https://github.com/ZcashFoundation/zebra/pull/8568 and we can go back to multi tread.